src/index.js

```diff
@@ -106,11 +106,11 @@
                 "required"      : true
             },
             {
-                "name"          : "accessKey",
-                "display_name"  : "Access Key",
+                "name"          : "token",
+                "display_name"  : "Jwt Token",
                 "type"          : "text",
-                "default_value" : (getCookie('DeviceHiveToken') || "AccessKeyExampleAccessKeyExampleAccessKeyEx="),
-                "description"   : 'Access key which is authorized to query device notifications. You can generate an access key using admin console by navigating to http://host/admin/#accesskeys or <a target="_blank" href="http://devicehive.com/restful#Reference/AccessKey/insert">by issuing a POST request</a>',
+                "default_value" : (getCookie('DeviceHiveToken') || "JwtTokenExampleJwtTokenExampleJwtTokenEx="),
+                "description"   : 'Jwt token which is authorized to query device notifications. You can generate a token using admin console by navigating to http://host/admin/#jwt-token or <a target="_blank" href="http://devicehive.com/restful#Reference/AccessKey/update">by issuing a POST request</a>',
```

Comment:
AccessKey reference will to be pointing to nowhere after updated rest api docs would be published